### PR TITLE
clusters/aaa: minor cleanup

### PR DESCRIPTION
**Pull request by spiffxp**
moved bucket creation for prowjobs into its own .tf file

rather than manually reformatting the resulting comments, I ran `terraform fmt` which updated a few more files in here as well

### CODE REVIEW

File: infra/gcp/clusters/projects/kubernetes-public/aaa/prowjob-buckets.tf [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  scalability_golang_builds_bucket_name = "k8s-infra-scale-golang-builds"    // Name of the bucket for the scalability golang builds
+}
+
+// Bucket for scalability tests results
+resource "google_storage_bucket" "scalability_tests_logs" {
+  project = data.google_project.project.project_id
+  name    = local.scalability_tests_logs_bucket_name
+
+  uniform_bucket_level_access = true
+}
```
Comment (ameukam) [inline infra/gcp/clusters/projects/kubernetes-public/aaa/prowjob-buckets.tf:32]: @spiffxp Can you add a lifecycle policy of 90 days of retention as suggested here : https://github.com/kubernetes/k8s.io/issues/1310#issuecomment-862440315? 🙏🏾

Comment (spiffxp) [reply infra/gcp/clusters/projects/kubernetes-public/aaa/prowjob-buckets.tf]: added in next commit

**Comment by ameukam** [comment]
/lgtm
/hold
Remove hold when ready to apply.

**Comment by spiffxp** [comment]
Ran `terraform apply`
```
google_storage_bucket.scalability_tests_logs: Modifying... [id=k8s-infra-scalability-tests-logs]
google_storage_bucket.scalability_tests_logs: Modifications complete after 1s [id=k8s-infra-scalability-tests-logs]

Apply complete! Resources: 0 added, 1 changed, 0 destroyed.
```

**Pull request merged**